apps/dashboard/src/@3rdweb-sdk/react/hooks/useBalanceForAddress.tsx

```diff
@@ -1,13 +1,23 @@
+import { thirdwebClient } from "@/constants/client";
 import { useQuery } from "@tanstack/react-query";
-import { useSDK } from "@thirdweb-dev/react";
+import { defineChain } from "thirdweb";
+import { getWalletBalance } from "thirdweb/wallets";
 import invariant from "tiny-invariant";
 import { walletKeys } from "../cache-keys";
 
-export function useBalanceForAddress(address: string | undefined) {
-  const sdk = useSDK();
+export function useBalanceForAddress(
```

Comment:
do we not have a hook for this exact thing in `thirdweb/react`? I fee like we *must* have one honestly.